### PR TITLE
handle encoded whitespace at moduleLoader.mjs

### PR DESCRIPTION
**Pull request by iambumblehead**

### CODE REVIEW

File: src/esmockLoader.mjs [hunk 1]
```diff
@@ -15,11 +15,12 @@ const urlDummy = 'file:///' + path
   .join(path.dirname(url.fileURLToPath(import.meta.url)), 'esmock.js')
   .replace(/^\//, '');
 
+const esmockGlobalsAndAfterRe = /\?esmockGlobals=.*/;
 const esMockGlobalsAndBeforeRe = /.*\?esmockGlobals=/;
 const esmockModuleKeysRe = /#esmockModuleKeys/;
 const exportNamesRe = /.*exportNames=(.*)/;
+const whitespaceRe = /%20/g;
```
Comment (aladdin-add) [inline src/esmockLoader.mjs:22]: not very familiar with the loader, but does it make sense to use `decodeURI("file://xxx")`, instead of hard-code `%20`?

Comment (iambumblehead) [reply src/esmockLoader.mjs:22]: great idea! thank you

**Comment by iambumblehead** [comment]
all tests are passing here I need to push your decodeURI suggestion to resolvewithplus and publish before the changes can appear here -- thank you @aladdin-add

**Comment by iambumblehead** [comment]
I pushed your suggested change to resolvewithplus and this branch. all tests are passing here.

I know tests should be added for all of these and I plan to add them in the next week or so but am tired to add them now,. do you want me publish this @aladdin-add?

**Comment by aladdin-add** [comment]
thanks, confirmed it was fixed. good to have it published if possible. 😄

**Comment by iambumblehead** [comment]
@aladdin-add its published thank you :)

**Pull request merged**